boot/freeldr/freeldr/include/arch/pc/x86common.h

```diff
@@ -12,10 +12,13 @@
 #define BIOSCALLBUFFER      HEX(4000) /* Buffer to store temporary data for any Int386() call */
 #define STACK16ADDR         HEX(6F00) /* The 16-bit stack top will be at 0000:6F00 */
 #define BSS_START           HEX(6F00)
-#define STACKLOW            HEX(7000)
+#define STACKLOW            HEX(8000)
 #define STACKADDR           HEX(F000) /* The 32/64-bit stack top will be at 0000:F000, or 0xF000 */
 #define FREELDR_BASE        HEX(F800)
 #define FREELDR_PE_BASE    HEX(10000)
+#define TEMPDATA16_BASE     HEX(7000)
```

Comment:
This value doesn't seem to be used in the code, so I suppose it's some sort of purely "informative" value, is it correct?
If so, what is being present between `TEMPDATA16_BASE` and `TEMPCODE16_BASE (- 1)` in memory?